pkgs/tools/networking/lychee/default.nix

```diff
@@ -7,16 +7,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "lychee";
-  version = "0.5.0";
+  version = "0.7.0";
 
   src = fetchFromGitHub {
     owner = "lycheeverse";
     repo = pname;
-    rev = "v${version}";
-    sha256 = "03dsp0384mwr51dkqfl25xba0m17sppabiz7slhxcig89b0ksykm";
+    rev = "f7f9485be0dc9435cee302cc49bb7f8a705fdff8";
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
Sure, I would prefer that as well, but the suggested code produces this.

```
copying path '/nix/store/bh3psvj27bp898kxmgbn8skxb7sx9508-git-2.31.1' from 'https://cache.nixos.org'...

trying https://github.com/lycheeverse/lychee/archive/v0.7.0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   126  100   126    0     0    402      0 --:--:-- --:--:-- --:--:--   402
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404
error: cannot download source from any mirror
builder for '/nix/store/m4c3zdc8p4wzi5z2nankmh1i6nd4kdfp-source.drv' failed with exit code 1
cannot build derivation '/nix/store/chhjhx5gfspmi5q934hq2nif7axmlikq-lychee-0.7.0.drv': 1 dependencies couldn't be built
error: build of '/nix/store/chhjhx5gfspmi5q934hq2nif7axmlikq-lychee-0.7.0.drv' failed
```

Reply:
```suggestion
    rev = version;
```
its actually

Reply:
:+1: thanks for the hint.